src/box/memtx_engine.cc

```diff
@@ -64,6 +64,7 @@
 #include "tt_sort.h"
 #include "assoc.h"
 #include "wal.h"
+#include "scoped_guard.h"
```

Comment:
Better do refactoring before the main patch.

Reply:
Done.